bitnami/haproxy/templates/NOTES.txt

```diff
@@ -0,0 +1,29 @@
+** Please be patient while the chart is being deployed **
+
+{{- if .Values.service.ports }}
+1. HAproxy has been started. You can find out the port numbers being used by HAProxy by running:
+
+          $ kubectl describe svc {{ include "common.names.fullname" . }} --namespace {{ .Release.Namespace }}
```

Comment:
The indentation of this code block is not consistent with below, could you check it?